templates/zerver/api/changelog.md

```diff
@@ -29,6 +38,12 @@ below features are supported.
 
 * [`GET /events`](/api/get-events): Removed `email` field from update
   subscription events.
+* Added two new events to [`GET /events`](/api/get-events) of type
+  stream_user_group_access. Out of the added event one has op=create
+  and the other one has op=delete. These events are added to
+  efficiently manage and deliver data related to user groups which
+  are allowed to post to streams which have post policy for allowing
+  only admins and selected user groups.
```

Comment:
Did you miss removing this while resolving conflicts? I see this block in both `Feature level 43` as well as `Feature level 44`.